features/other_b/support.feature

```diff
@@ -44,3 +44,16 @@ Feature: Filing a support request
     And I press "Send"
   Then I should see "Your message was sent to the Archive team - thank you!"
     And 2 emails should be delivered
+
+  Scenario: Invalid form submission
+
+  When I am on the home page
+    And basic languages
+    And I follow "Support and Feedback"
+  When I select "Deutsch" from "feedback_language"
+    And I fill in "Brief summary" with "Just a brief note"
+    And I fill in "Your comment" with "Men have their old boys' network, but we have the OTW. You guys rock!"
+    And I fill in "Your email (required)" with "a"
+    And I press "Send"
+  Then I should see "Sorry, your message could not be saved"
+    And the field labeled "feedback_language" should contain "Deutsch"
```

Comment:
Instead of adding a new scenario you could also build off the existing one ("Scenario: Not logged in, with and without email", which has an error on submission part), the way you added to the abuse report scenario.